src/deepsparse/pipeline.py

```diff
@@ -49,6 +54,51 @@
 _REGISTERED_PIPELINES = {}
 
 
+def question_answering_pipeline(*args, **kwargs) -> "Pipeline":
+    """
+    :param args: the positional arguments to pass into Pipeline.create
+    :param kwargs: the keyword arguments to pass into Pipeline.create
+    :return: question_answering pipeline created with args and kwargs
+    """
+    return Pipeline.create(*(("question_answering",) + args), **kwargs)
+
+
+def text_classification_pipeline(*args, **kwargs) -> "Pipeline":
```

Comment:
let's make another one called `sentiment_analysis_pipeline` that does the same thing